tests/e2e/support/environment/sse.ts

```diff
@@ -0,0 +1,50 @@
+import { fetchEventSource, EventStreamContentType } from '@ai-zen/node-fetch-event-source'
+import { getAuthHeader } from '../api/http'
+import { User } from '../types'
+
+const sseEventStore = {}
+const ctrl = new AbortController()
+
+export const listenSSE = (baseUrl: string, user: User): Promise<void> => {
```

Comment:
I couldn't find anything to listen to the sse events in browser using playwright. So what I have tried is, listen to the sse event by each user after login separately (using `@ai-zen/node-fetch-event-source`), collect the events for each user and check the events